webapp/src/modules/analytics/track.js

```diff
@@ -104,12 +105,15 @@ add(
   action => action.payload
 )
 
-add(MANAGE_PARCEL_SUCCESS, 'Manage LAND Permissions', action => ({
-  x: action.parcel.x,
-  y: action.parcel.y,
-  address: action.address,
-  revoked: action.revoked
-}))
+add(
+  MANAGE_ASSET_SUCCESS,
+  action => addAssetType('Manage Asset Permissions', action.asset_type),
```

Comment:
This will end up as `Manage Asset Permissions Parcel` and `Manage Asset Permissions Estate`, doesn't sound better to just be called something like `Manage Parcel` and `Manage Estate` ?